php/libraries/Database.class.inc

```diff
@@ -28,6 +31,16 @@ define("DEBUG", false);
  */
 class Database
 {
+    /**
+     * Constants determining whether HTML characters will be escaped before
+     * being inserted into the Database.
+     *
+     * @access private
+     */
+    const ESCAPE_ALL      = 0;
+    const ESCAPE_NOQUOTES = 1;
+    const ESCAPE_NONE     = 2;
```

Comment:
Are these const needed here?
all are already DEFINEd and no use of $this->CONST is made.

Reply:
Yep, they are left over from a previous approach. Will delete.